recipe/bld.bat

```diff
@@ -22,8 +22,8 @@ cmake -G"NMake Makefiles" ^
       -DUSE_SUITESPARSE=0 ^
       -DUSE_FOX=0 ^
       -DBUILD_FOX=0 ^
-      -DUSE_FFTW=1 ^
-      -DBUILD_ZLIB_AND_LIBPNG=1 ^
+      -DUSE_FFTW=0 ^
+      -DBUILD_ZLIB_AND_LIBPNG=0 ^
```

Comment:
BUILD_ZLIB_AND_LIBPNG should be 1 under Windows, if you want to have USE_PNG=1.

Reply:
Yes, sorry, I have just noticed it and missed that one! I was checking locally before pushing the change.